pyroscope_cli/Cargo.toml

```diff
@@ -34,10 +34,10 @@ slog-term = "2.8.0"
 slog-scope = "4.4.0"
 slog-async = "2.7.0"
 slog-stdlog = "4.1.0"
-pyroscope = { version = "0.5.4", path = "../" }
-pyroscope_pprofrs = {version = "0.2.3", path = "../pyroscope_backends/pyroscope_pprofrs" }
-pyroscope_rbspy = {version = "0.2.2", path = "../pyroscope_backends/pyroscope_rbspy" }
-pyroscope_pyspy = {version = "0.2.3", path = "../pyroscope_backends/pyroscope_pyspy" }
+pyroscope = { version = "0.5.4", path = "../", default-features = false }
```

Comment:
we need to commit lock files for cli, ffi/ruby, ffi/python

Reply:
let me know if I updated all of these correctly